.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: 🤔 Question
+    url: https://mixxx.org/forums
+    about: Please use our forums for questions

```

Comment:
> I don't think thats possible.

It's possible to add links (with title and description) as "contact links". What do you think about this @Holzhaus?
Remember that you can try this at https://github.com/MatteoGheza/mixxx-issue-templates-PoC/issues/new/choose

Reply:
Great, exactly what I had in mind! Thanks.

Reply:
Good catch. I didn't know there was an `ISSUE_TEMPLATE/config.yml`. I was only digging through the documentation of the forms yaml scheme. 
This solution is much better than my proposal.

Reply:
Nitpick: instead of arbitrary `Open` let's use `Go to Mixxx user forum`?

Reply:
if possible at all...

Reply:
I think this is not possible... The config file has not more options than this.